src/styles/app.css

```diff
@@ -0,0 +1,27 @@
+.app {
+  margin: 20px;
+  height: 100vh;
```

Comment:
This causes a bit of a bug, when scrolling down the page the background image disappears. If this is intentional then ignore me, if not you fix it by replacing this with `min-height: 100vh`